.github/workflows/withAptitude.yml

```diff
@@ -0,0 +1,110 @@
+# Not cross-platform, but works
+
+name: withAptitude
+
+# Run this workflow manually from the Actions tab
+on: workflow_dispatch
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "populate"
+  populate:
+    # Run on a single platform or use a matrix to test against several
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [ '3.8'  ]
+    env:
+       TMPTMPDIR: /tmp
+       OS_CHOICE: ${{ matrix.os }}
+       
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    name: gr-bokeh on ${{ matrix.os }} with python ${{ matrix.python-version }} 
+    steps:
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: x64
+       
+      - name: Make a place to store logs
+        run:  mkdir -p ~/artifacts
+
+      - name: Security first
+        run: |
+          mkdir ~/logs
+          sudo apt update
+          sudo apt install -y software-properties-common apt-utils 
+          sudo apt install -y unattended-upgrades
+          sudo unattended-upgrade -d >~/artifacts/securityFirst.log
+
+      - name: Ensure third-party apt support is installed
+        run: |
+          if [ "${OS_CHOICE}" = "ubuntu-latest" ]; then
+             sudo add-apt-repository -y ppa:gnuradio/gnuradio-releases-3.9
+          fi
+          sudo apt update
+
+      - name: Install the tangled web 
+        run: |
+          sudo apt install -qy build-essential npm nodejs cmake cmake-curses-gui ninja-build python3-pip
+          sudo apt install -qy lrzip cmake g++ libboost-all-dev libgmp-dev swig python3-numpy \
```

Comment:
Swig is not needed anymore starting with gnuradio 3.9.

Reply:
Noted.  I will assume the master branch are as per the README: GNU Radio 3.9.5 or 3.10, Bokeh library above 2.3.1,
NodeJS >= 14.  I will test against both versions of GNUradio, Bokeh 2.3.1 and 2.4.2, and the latest npm/node available as of test deployment.

Reply:
That sounds good